### ISSUE TITLE
script fails on titles that start with <emph> tag

### ISSUE THREAD

**Issue by ngeraci** [ngeraci/ucr_archivestools#12]
Bug report from Jessica via email:

I’m trying to publish the EAD for MS 051, the Biblioteca del Niño Mexicano facsimilies collection, and I’m having an error with the script. I downloaded the new updates, and they worked fine last week so I’m not sure if there’s something I need to fix in the ASpace record or not. Here’s what the error message is:

```
$ oac-process "C:\Users\jgeiser\Downloads\MS.051_20200224_212724_UTC__ead.xml"
Traceback (most recent call last):
  File "C:\Users\jgeiser\AppData\Local\Continuum\miniconda3\Scripts\oac-process-script.py", line 11, in <module>
    load_entry_point('aspace-tools==0.3', 'console_scripts', 'oac-process')()
  File "C:\Users\jgeiser\AppData\Local\Continuum\miniconda3\lib\site-packages\aspace_tools-0.3-py3.6.egg\aspace_tools\oac_process.py", line 69, in main
    finding_aid.process()
  File "C:\Users\jgeiser\AppData\Local\Continuum\miniconda3\lib\site-packages\aspace_tools-0.3-py3.6.egg\aspace_tools\oac_process.py", line 204, in process
    lxml_operations(self)
  File "C:\Users\jgeiser\AppData\Local\Continuum\miniconda3\lib\site-packages\aspace_tools-0.3-py3.6.egg\aspace_tools\oac_process.py", line 133, in lxml_operations
    titleproper.text = titleproper.text.strip()
AttributeError: 'NoneType' object has no attribute 'strip'
```

**Comment by ngeraci** [comment]
* Finding aid title is `<emph>Biblioteca del Niño Mexicano</emph> facsimiles collection`
* Tested on MS 426 `<emph>Brown Girl Begins</emph> collection` and got same error
* Tested on MS 032 `Sue Mitchell collection on <emph render="italic">52</emph>` (`emph` tag at end of title) -- no error

**Comment by ngeraci** [comment]
fixed by 2eab4caa
